clientcontroller/babylon/babylon.go

```diff
@@ -28,14 +30,16 @@ import (
 	"go.uber.org/zap"
 	protobuf "google.golang.org/protobuf/proto"
 
+	"github.com/babylonlabs-io/finality-provider/clientcontroller/api"
 	fpcfg "github.com/babylonlabs-io/finality-provider/finality-provider/config"
 	"github.com/babylonlabs-io/finality-provider/types"
 )
 
-var _ ClientController = &BabylonController{}
+var _ api.ClientController = &BabylonController{}
 
 var emptyErrs = []*sdkErr.Error{}
 
+//nolint:revive
```

Comment:
what does this mean?

Reply:
It's a linter rule for some naming consistency check. If we remove this line then linter will fail with the following

```
➜  finality-provider git:(base/consumer-chain-support) golangci-lint run 
clientcontroller/babylon/babylon.go:42:6: exported: type name will be used as babylon.BabylonController by other packages, and that stutters; consider calling this Controller (revive)
type BabylonController struct {
     ^
```

For now we surpassed such naming convention issues. We can revise naming afterwards. Wdyt?

Reply:
Good idea